src/nls/messages_en.json

```diff
@@ -17,6 +22,11 @@
     "dashboard.errorBoundary.defaultError": "Something went wrong",
     "dashboard.errorBoundary.pageError": "Error loading page",
     "dashboard.header.logOut": "Log out",
+    "dashboard.keyValueList.addButton.ariaLabel": "Add Button. Adds {legendText} entry.",
+    "dashboard.keyValueList.deleteButton.ariaLabel": "Delete Button#{index}. Deletes entire entry in current text field.",
```

Comment:
I'm still struggling a bit with these strings.

'Delete Button#0', will this make sense to a user?
'current text field', how do I know which is the 'current' text field?

Reply:
You're right, those strings don't really make sense. Since there's already text on the screen for each of these elements, I'm going to remove the aria labels. Is that alright?